.travis.yml

```diff
@@ -0,0 +1,24 @@
+
+language: python
+
+python:
+  - "3.6"
+
+jobs:
+  include:
+    - name: test
+      services:
+        - xvfb
+      before_script:
+        - "export DISPLAY=:99.0"
```

Comment:
I'm surprised not to see `export TRAVIS=true` anywhere? Why isn't that needed?

Reply:
That is not needed as Travis internally provides an environment variable named `TRAVIS` with value `'true'`. This is we are using in our script.